x-pack/plugin/watcher/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/watcher/execute_watch/90_auto_create_index.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
I like this test case, but I think it should be moved to the `put_watch` directory, rather than being in `execute_watch`.